telemetry/tracelog/src/main/scala/io/buoyant/telemetry/tracelog.scala

```diff
@@ -0,0 +1,46 @@
+package io.buoyant.telemetry
+
+import com.twitter.finagle.Stack
+import com.twitter.finagle.buoyant.Sampler
+import com.twitter.finagle.stats.NullStatsReceiver
+import com.twitter.finagle.tracing.{Trace, Record, TraceId, Tracer}
+import com.twitter.logging.{Level, Logger}
+
+class TracelogInitializer extends TelemeterInitializer {
```

Comment:
It might be more consistent with other initializers to call this file TracelogInitializer.scala, instead of tracelog.scala.